scripts/coverage_report_jenkins.py

```diff
@@ -2,20 +2,44 @@
 # -*- coding: utf-8 -*-
 import argparse
 import jenkins
+import os
 import re
 import requests
 import subprocess
+import time
 
 from requests.auth import HTTPBasicAuth
 from urlparse import urlsplit, urlunsplit
 
 from cfme.utils.appliance import IPAppliance
+from cfme.utils.conf import env
 from cfme.utils.path import log_path
 from cfme.utils.quote import quote
 from cfme.utils.version import Version
 
+coverage_dir = '/coverage'
+scan_timeout = env.sonarqube.scan_timeout
```

Comment:
Consider using `env.sonarqube.get('scan_timeout', <default>)` so that if sonarqube is in the yaml, but scan_timeout isn't set there, this doesn't fail with AttributeException. Same for `url` and `scanner_url` below, though those can't really be defaulted.

Reply:
Changed scan_timeout, did not change the the url ones because as you said there really isn't a reasonable timeout.